### ISSUE TITLE
please wait of heck

### ISSUE THREAD

**Issue by gordonwoodhull** [att/rcloud#1785]
I'm getting the "please wait of heck" even when I'm quite sure I'm not interrupting anything.

I'm tempted to get rid of the "curtain" entirely, but there is another bug here. Debug console reports:

```
WebSocket connection to 'ws://135.197.192.220:8080/edit.html?notebook=96e762bea9897aa21427&version=59752b89df772fc8fc239cbaea8a8783eef97b52' failed: WebSocket is closed before the connection is established.
```

Steps to reproduce:
 1. Load notebook
 2. "Undo" to old version (or skip step 1 and start on old version)
 3. Wait as long as you want
 4. "Redo" to current version

**Comment by gordonwoodhull** [comment]
This is the first time I've been sure that I'm not interrupting one load with another, but it makes me think that some other cases must also be unrelated.

Hopefully we can fix all of these for 1.6. Super annoying.

**Comment by gordonwoodhull** [comment]
The code was executing two load commands at once, due to the ambiguity of cmd-Z and cmd-opt-Z.

We should probably be more defensive about checking that more modifier keys are *not* being pressed, especially as we start to use more keys. It's also rude because extensions may want to grab key combinations that we've inadvertently stolen.
